src/router.js

```diff
@@ -1,4 +1,5 @@
 import Router from 'vue-router';
+import store from './store';
```

Comment:
Don't import store explicitly to don't block SSR because there each request has an own store and router https://github.com/aeternity/superhero-ui/issues/909